import "session_commands.proto";

message Command_DeckDownload {
	extend SessionCommand {
		optional Command_DeckDownload ext = 1012;
	}
	optional uint32 deck_id = 1;
}
 
